libopendroneid/opendroneid.h

```diff
@@ -458,6 +459,102 @@ float decodeVerticalAccuracy(ODID_Vertical_accuracy_t Accuracy);
 float decodeSpeedAccuracy(ODID_Speed_accuracy_t Accuracy);
 float decodeTimestampAccuracy(ODID_Timestamp_accuracy_t Accuracy);
 
+// OpenDroneID WiFi functions
```

Comment:
Would it make sense to separate this out to a separate wifi.h header file and then also move the wifi.c file to the wifi directory?